content/tutorials/generic/bootloader-reloading/bootloader-reloading.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Burn the bootloader on non-AVR boards
+description: Learn how to burn the bootloader on non-AVR boards.
+difficulty: Advanced
+featuredImage: 'clip'
+---
+
+In this tutorial, we will learn how to burn the bootloader on non-AVR boards using another Arduino board. The bootloader is a small piece of software that allows uploading sketches onto the Arduino board. It comes preprogrammed on the microcontrollers on Arduino boards. Whether the bootloader has been corrupted or intentionally has been removed, it can be restored by burning (also called, flashing or programming) a new bootloader to the board.
+
+>**Note:** If you want to burn the bootloader on AVR boards such as an UNO, Mega, and classic Nano you can check out [these instructions](https://support.arduino.cc/hc/en-us/articles/4841602539164-Burn-the-bootloader-on-UNO-Mega-and-classic-Nano-using-another-Arduino).
+
+## Requirements
+- An extra Arduino board that runs at 3.3V to use as the programmer.
+
+>**Note:** Certain Arduino boards can't be used with the sketch that converts it to a programmer. It is possible to use an Arduino board that runs at 5V as the programmer, but you will need to use level shifting circuitry on the programming lines to avoid exposing the target board to 5V logic levels, which would damage it.
+
+| Working | Untested | Not working |
+| ----------- | ----------- |----------- |
+| SAMD architecture boards | AVR architecture boards | Nano 33 BLE |
+
```

Comment:
```suggestion
- An extra SAMD-based Arduino board to use as the programmer.
```

In the context of a forum thread, it might have been reasonable to start a discussion about which boards might be used as a programmer, but in the context of a tutorial, that is inappropriate. The tutorial should be focused on a known working procedure the reader can follow.

I do think it would be worthwhile, and also quite a fun assignment, to take the time to test whether other official Arduino boards might be used as a programmer. I verified that the example sketch is still failing to compile for the **Nano 33 BLE** board, so that is out, but it does compile for **Nano RP2040 Connect** and **Portenta H7 (M7 core)**, which I think are both interesting programmer candidates. It also compiles for **Mega** (other AVR boards don't have enough memory for the sketch), but I think that is a less valuable programmer candidate for the purposes of the tutorial due to the need to do logic level translation between the 5 V Mega and the 3.3 V SAMD board.

If other boards are verified to work, they could be added to a list here (I would still avoid adding a list of non-working or untested boards).